evergreen/evergreen.yml

```diff
@@ -596,9 +596,6 @@ tasks:
         # - func: upload-build
 
     - name: test
```

Comment:
Should we also remove dependency from [plain-auth-tests](https://github.com/vincentkam/mongo-csharp-driver/blob/50b77640f3a2c729f4f6306a7cd473e6ea824835/evergreen/evergreen.yml#L610), [aws-auth-tests](https://github.com/vincentkam/mongo-csharp-driver/blob/50b77640f3a2c729f4f6306a7cd473e6ea824835/evergreen/evergreen.yml#L617) and [atlas-connectivity-tests](https://github.com/vincentkam/mongo-csharp-driver/blob/50b77640f3a2c729f4f6306a7cd473e6ea824835/evergreen/evergreen.yml#L603)?

Reply:
Ah, a reasonable idea! Thanks! I've taken your suggestion :)